src/features/advanced-content-insights/messages.ts

```diff
@@ -17,6 +17,12 @@ const messages = defineMessages({
         description: 'Description text about advanced content insights.',
         id: 'boxui.advancedContentInsights.advancedContentInsightsDescription',
     },
+    advancedContentInsightsTooltipText: {
```

Comment:
since this is the same as `advancedContentInsightsDescription`, do we need separate messages? do we anticipate that the messages will diverge in the future?